### PR TITLE
warning if snapshot too large

### PR DESCRIPTION
**Pull request by zhenz**
Changes include:
1. warn user if tar is larger than 8G and ask if continue
2. have a properties section in the shell script to avoid copy-and-pastes

### CODE REVIEW

**Comment by zhenz** [comment]
@priteau Please review! Thanks!

**Comment by zhenz** [comment]
Sorry, @priteau , but can you hold on reviewing the code. I was trying to avoid copy-and-pastes in the code by creating a property file, but I might complicate things.  Will let you know when I'm done. Thanks!

**Comment by zhenz** [comment]
@priteau Done fixing. I was trying to create a properties file for all properties, but to make it work I have to change CC-CentOS and CC-Ubuntu repos as well. So I gave that up and just modified the shell script. Please review! Thanks!

File: cc-snapshot [hunk 1]
```diff
@@ -8,6 +8,29 @@ if [[ $EUID -ne 0 ]]; then
    exit 1
 fi
 
+# ===============================================================================
+## Properties
+# ===============================================================================
+## OpenStack Properties
+# ===============================================================================
+OPENSTACK_VENDER_DATA='http://169.254.169.254/openstack/latest/vendor_data.json'
```
Comment (priteau) [inline cc-snapshot]: s/VENDER/VENDOR

File: cc-snapshot [hunk 2]
```diff
@@ -98,14 +121,8 @@ fi
 export OS_REGION_NAME=$REGION
 if [ -z "$OS_REGION_NAME" ]; then unset OS_REGION_NAME; fi
 
-# if [ $KEYSTONE_VERSION = '2' ]; then
-export OS_AUTH_URL=https://chi.$SITE.chameleoncloud.org:5000/v2.0
+export OS_AUTH_URL=$(echo $CHAMELEON_OS_AUTH_URL_TMPL | sed "s/SITE/$SITE/g")
 export OS_TENANT_ID=$PROJECT_ID
-# else #keystone v3 ...migrate to this at some point
```
Comment (priteau) [inline cc-snapshot]: Please keep the v3 code as we will likely move to it rather than keep using v2.0

**Review by priteau — CHANGES_REQUESTED** [review]

**Comment by zhenz** [comment]
@priteau Done fixing.

**Review by priteau — APPROVED** [review]

**Pull request merged**